JobToResume.md

```diff
@@ -0,0 +1,69 @@
+Job To Resume
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+Job To Resume Service recommends resumes based on job data. Job To Resume Service is available at 
+`/core/jobtoresume`.
+
+
+## Request Structure
+Requests consist of:
+
+|                   |               |
+|-------------------|---------------|
+|`carotene_id`      | string        | 
+|`carotene_version` | string        |
+|`skills`           | List(Skill)   |
+|`location`         | Location      |
```

Comment:
Don't we want to add a brief description of what each parameter does or is used for?

Reply:
In this table, I mean.

Reply:
Also, this is JSON, so there are no Lists; refer to them as Arrays instead.

Reply:
And the Location object structure would probably be needed to be detailed as well, unless we can relax and let them discover it fully from the swagger docs or the generated SDK. Thoughts, anyone?

Reply:
On a second thought, scratch the comment on the Location: the example is probably already pretty clear on what to put in a location.

Reply:
we are giving a sample curl request which will give user a clear idea about the fields,thats why I didn't explain.If you think we need explanation,I can add.Thoughts?

Reply:
Yes, I think the descriptions may still be needed: are all parameters required? Are some of them optional, what happens if omitted? You already had to clarify what are the acceptable values for `carotene_version`, perhaps that sentence belonged to the description of the param.

Reply:
ok